### ISSUE TITLE
#ifdef notdef

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/xania#19]
`#ifdef notdef` is in some parts of the code. We should either expose the hidden code or delete it.

**Comment by snellers** [comment]
I removed the notdef in db.c line 1512, and it reveals a considerable list of items that need tuning relative to their owners.  I think that block of code it pretty useful so we should keep it. I will log another issue to cover tuning these items.  Then we can enable the block generally to trap problematic items earlier in future.

The 2nd block is the feature that randomizes a room's exits every time its area resets (approx every 15 minutes), _if_ the room in the RESETS section has been configured to use an 'R' random flag.  This is ifdef'd out, yet we have at last 15 areas that use this feature.  I really don't know how/why this happened. I did some testing and the randomization does work.  And, when enabled, we don't get any new errors on startup, so I'm going to re-enable this feature.  Side note: we do get Fix_exits errors on startup, but I'm fairly sure these are unrelated.
